ext/rubymain.cpp

```diff
@@ -255,6 +255,14 @@ static VALUE t_run_machine (VALUE self UNUSED)
 	return Qnil;
 }
 
+/*****************************
+t_get_outstanding_timer_count
+*****************************/
+
+static VALUE t_get_outstanding_timer_count ()
+{
+    return SIZET2NUM (evma_get_outstanding_timer_count ());
```

Comment:
I was about to write that I don't think `size_t` can be converted to a Ruby type. Apparently there is a macro for that!

Reply:
Not available in Ruby 1.8.7. Time to drop Ruby 1.8.7!

Reply:
Hmm. I can remove 1.8.7 support, or I could fudge this by converting `size_t` to some kind of unsigned int. What's your preference? I prefer removing support - 1.8.7 is very old, end-of-life was 4 years ago, and we don't use it anywhere at Pusher.

Reply:
I've gone ahead and removed 1.8.7 support